includes/amp-helper-functions.php

```diff
@@ -607,9 +607,13 @@ function amp_get_slug() {
  */
 function amp_get_current_url() {
 	$parsed_url = wp_parse_url( home_url() );
+
 	if ( ! is_array( $parsed_url ) ) {
-		$parsed_url = [];
+		$parsed_url = [
+			'host' => 'localhost',
+		];
```

Comment:
@westonruter Oh wait, what about the condition below that gets `host` from `$_SERVER['HTTP_HOST']`?

https://github.com/ampproject/amp-wp/blob/91efb9772d002355a84950a6fb0cee1dd5e46bce/includes/amp-helper-functions.php#L622-L624

Reply:
Good point. Then it should be set to an empty array here, and below if HTTP_HOST is not defined, then use localhost.

Reply:
Done in https://github.com/ampproject/amp-wp/pull/5219/commits/296d0b1fc3eded1a7bbbc6a8aef919254c44d33d.